.github/workflows/ci.yml

```diff
@@ -1,22 +1,60 @@
-name: CI
+name: camino wallet ci
 
 on:
-  push:
```

Comment:
I'm against this: CI should run on every branch / every push so devs can see without PR if their changes are fine